quickstart/201-vm-disk-encryption-extension/main.tf

```diff
@@ -130,3 +134,18 @@ resource "azurerm_virtual_machine_extension" "example" {
     "VolumeType"             = "All"
   })
 }
+
+resource "local_sensitive_file" "lsf" {
+  content  = tls_private_key.vm_key.public_key_openssh
+  filename = "id_rsa"
```

Comment:
Please rename filename to `key.pem` as we'll export private key in pem format, thanks.